lib/facebook/messenger/version.rb

```diff
@@ -1,5 +1,5 @@
 module Facebook
   module Messenger
-    VERSION = '0.10.0'.freeze
+    VERSION = '0.11.0'.freeze
```

Comment:
This will definitely be `0.11.0`, but best leave it to the maintainer to bump the version number in the future. They might want to [do it in a separate commit](https://github.com/hyperoslo/facebook-messenger/commit/8660f6dc645e33fd9b71c31d659cdb436ca55b8e), for example. ;-)